modules/tests/src/smithy4s/tests/PizzaSpec.scala

```diff
@@ -440,11 +440,13 @@ abstract class PizzaSpec
         )
         val containsAllExpectedHeaders =
           expectedHeaders.forall(h => headers.get(h._1).contains(h._2))
+        val missingHeadersMessage =
+          s"Expected to find all of $expectedHeaders inside of $headers"
```

Comment:
This extraction helps avoid false positive error that started happening after updating weaver:

```
[info] [tests] compiling 5 Scala sources to /home/majk/Code/personal/smithy4s/modules/tests/target/jvm-2.13/classes ...
[error] [tests] /home/majk/code/smithy4s/modules/tests/src/smithy4s/tests/PizzaSpec.scala:432:15: possible missing interpolator: detected interpolated identifiers `$expectedHeaders`, `$headers`
[error] [tests]       } yield {
[error] [tests]               ^
[info] [bootstrapped] compiling 1 Scala source to /home/majk/code/smithy4s/modules/bootstrapped/target/jvm-2.13/classes ...
[error] [tests] No warnings can be incurred under -Werror.
[error] [tests] two errors found
[error] [tests] (tests / Compile / compileIncremental) Compilation failed
```